### DISCUSSION TITLE
Setting up RAID1 array & migrating data. | dinhcap's blog

### DISCUSSION THREAD

**Discussion by giscus** [dng-nguyn/dinhcap-dev#2] [General]
# Setting up RAID1 array & migrating data. | dinhcap's blog

Setting up RAID1 in Linux for increased data reliability.

http://localhost:1313/posts/raid-setup/

**Comment by dng-nguyn** [comment]
This is a test comment

**Comment by dng-nguyn** [comment]
This is another

**Comment by chinh1402** [comment]
This site has so much traffic oh my gawd
